docs/articles/infrastructure/ci-cd/jenkins.rst

```diff
@@ -1,7 +1,3 @@
-:orphan:
```

Comment:
Don't remove the orphan, because without it any other project that doesn't use this file won't build.